.announce

```diff
@@ -1,44 +0,0 @@
-#!/bin/bash -e
```

Comment:
I think this file is used to update version in README.md and on the ktlint.github.io site.

Reply:
Reverted